solutions/fully-configurable/variables.tf

```diff
@@ -16,7 +16,7 @@ variable "existing_secrets_manager_crn" {
 
 variable "prefix" {
   type        = string
-  description = "The prefix to add to all resources created by this solution. To not use any prefix value, you can set this value to `null` or an empty string."
+  description = "The prefix to be added to all resources created by this solution. To skip using a prefix, set this value to null or an empty string. The prefix must begin with a lowercase letter and may contain only lowercase letters, digits, and hyphens '-'. It should not exceed 16 characters, must not end with a hyphen('-'), and can not contain consecutive hyphens ('--'). Example: prod-0205-cos."
```

Comment:
@Aayush-Abhyarthi Not sure `prod-0205-cos` is a good example - there is no COS deployed by this DA. Perhaps `prod-us-south` is better